test/client/fixtures/sandbox/shadow-ui-test.js

```diff
@@ -537,6 +537,28 @@ test('querySelectorAll', function () {
     strictEqual(elems[0].id, 'pageElem');
 });
 
+test('document.querySelector and head.querySelector', function () {
+    var meta       = document.createElement('meta');
+    var shadowMeta = document.createElement('meta');
+
+    meta.className       = 'metas';
+    shadowMeta.className = 'metas ' + SHADOW_UI_CLASSNAME.charset;
```

Comment:
It will add one `metas-hammerhead-shadow-ui` class, but I need to add two class `metas` and `SHADOW_UI_CLASSNAME.charset`